Administrator Guide/Split brain and ways to deal with it.md

```diff
@@ -0,0 +1,68 @@
+# Split brain and the ways to deal with it
+
+### Split brain:
+Split brain is a situation where two or more replicated copies of a file become divergent. When a file is in split brain, there is an inconsistency in either data or metadata of the file amongst the bricks of a replica and do not have enough information to authoritatively pick a copy as being pristine and heal the bad copies, despite all bricks being up and online. For a directory, there is also an entry split brain where a file inside it can have different gfid/file-type across the bricks of a replica. Split brain can happen mainly because of 2 reasons:
+1. Due to network disconnect:
+Where a client temporarily loses connection to the bricks.

```

Comment:
For 2. Below there is a good example. May be we should also explain how two different clients connecting to two different bricks in the replica pair at the same time because of network split would help too?